src/client/src/orangehrmAdminPlugin/pages/ldap/LdapConfiguration.vue

```diff
@@ -160,34 +160,49 @@
               </oxd-text>
             </oxd-grid-item>
             <oxd-grid-item class="--offset-row-4">
+              <oxd-input-field
+                v-model="configuration.userSearchFilter"
+                label="User Search Filter"
```

Comment:
TODO:: i18n

Reply:
Will add a PR to your branch with these

Reply:
https://github.com/RajithaKumara/orangehrm/pull/48

Reply:
Merged. Thanks :relaxed: